src/components/Button/index.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 
 interface ButtonProps {
   name: string;
+  dataTestId?: string;
   disabled?: boolean;
   className?: string;
 }
 
-const Button = ({ name, disabled, className }: ButtonProps): JSX.Element => {
+const Button = ({ name, dataTestId, disabled, className }: ButtonProps): JSX.Element => {
```

Comment:
If you agree with the previous comment, you can change this to:
```suggestion
const Button = ({ name, disabled, className, ...htmlAttributes }: ButtonProps): JSX.Element => {
```

Reply:
I've changed it to use ...props to be like other components 😀